packages/create-plugin/templates/github/workflows/is-compatible.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       - name: Build plugin
         run: {{ packageManagerName }} run build
       - name: Compatibility check
-        run: npx @grafana/levitate@latest is-compatible --path src/module.ts --target @grafana/data,@grafana/ui,@grafana/runtime
+        run: npx @grafana/levitate@latest is-compatible --path src/module.{{#if isAppType}}tsx{{else}}ts{{/if}} --target @grafana/data,@grafana/ui,@grafana/runtime
```

Comment:
what about using 
```suggestion
        run: npx @grafana/levitate@latest is-compatible --path $(find . -regex "./src/module\.tsx?") --target @grafana/data,@grafana/ui,@grafana/runtime
```

Reply:
Yeah, I think something like this could actually be better! Unfortunately this regex doesn't work though - GPT was suggesting the same :D What about something like `find ./src -type f \( -name "module.ts" -o -name "module.tsx" \)`?

Reply:
(Something like this would be actually better indeed as the code would keep working even if we change the extensions for the other plugin types, too.)

Reply:
your suggestion also works (tested). regex works for me but I guess is a linux/mac problem.

Reply:
> regex works for me but I guess is a linux/mac problem.

Yeah, that must be it. Updated in https://github.com/grafana/plugin-tools/pull/1384/commits/cc973b364b557963f0864f6acffa55ba69292e5e.